lib/rules/space-infix-ops.js

```diff
@@ -8,6 +8,23 @@
 // Rule Definition
 //------------------------------------------------------------------------------
 
+const OPERATORS = [
```

Comment:
My suggestion would be to see if a utility (e.g., ast-utils) could be augmented with this information, so at least multiple rules could benefit. But I have no issue with it being inside the rule for now.